index.html

```diff
@@ -205,87 +240,87 @@ <h2 class="card-title">Tonic</h2>
-            /></a>
-          </li>
-          <li>
-            <a href="https://twitter.com/HFG__43"
-              ><img src="Images_Background/Twitter.svg" alt="Twitter Icon"
-            /></a>
-          </li>
-          <li>
-            <a href="#"><img src="Images_Background/M.svg" alt="M Icon" /></a>
-          </li>
-        </ul>
-        <div class="button-container principal-margin">
-          <button class="see-project-button">Get my resume</button>
-        </div>
-        <ul class="about-myself-skills principal-margin">
-          <li class="arrow-skills">
-            <p>Languages</p>
-            <i class="fa fa-angle-down arrow-color"></i>
-          </li>
-          <ul class="tech-languages">
+          </p>
+          <h2 class="lets-connect principal-margin-about">LET'S CONNECT</h2>
+          <ul class="social-icons social-icons-margin-bottom-about-myself principal-margin-about-social">
+            <li>
+              <a href="https://github.com/HFG43"
+                ><img src="Images_Background/GitHub.svg" alt="GitHub Icon"
+              /></a>
+            </li>
             <li>
-              <img src="Images_Background/JS-logo.svg" alt="JavaScript logo" />
-              <p class="language-name">JavaScript</p>
+              <a
+                href="https://www.linkedin.com/in/hern%C3%A1n-g%C3%BCemes-a440591b/"
+                ><img
+                  src="Images_Background/Linkedin icon.svg"
+                  alt="LinkedIn Icon"
+              /></a>
             </li>
             <li>
-              <img src="Images_Background/html-logo.svg" alt="HTML logo" />
-              <p class="language-name">HTML</p>
+              <a href="#"
+                ><img src="Images_Background/Fingers.svg" alt="Fingers Icon"
+              /></a>
             </li>
             <li>
-              <img src="Images_Background/css-logo.svg" alt="CSS logo" />
-              <p class="language-name">CSS</p>
+              <a href="https://twitter.com/HFG__43"
+                ><img src="Images_Background/Twitter.svg" alt="Twitter Icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="Images_Background/M.svg" alt="M Icon" /></a>
             </li>
           </ul>
-          <li class="bottom-line-language arrow-skills">
-            <p>Frameworks</p>
-            <i class="fa fa-angle-down arrow-color"></i>
-          </li>
-          <li class="bottom-line-language arrow-skills">
-            <p>Skills</p>
-            <i class="fa fa-angle-down arrow-color"></i>
-          </li>
-        </ul>
+          <div class="button-container principal-margin-about">
+            <button class="see-project-button">Get my resume</button>
+          </div>
+        </div> 
+        <div class="about-myself-skills-section">
+          <ul class="about-myself-skills principal-margin">
+            <li class="arrow-skills">
+              <p>Languages</p>
```

Comment:
- [ ] your design as shown below for this section is not consistent wit the figma template, there needs to be spacing between each technology and the arrows for frameworks and skills are not pointing towards the right direction. please kindly fix all these minor issues because they show that you pay alot of attention to details when implementing figma design :+1: 
- figma design
![l-d-f](https://user-images.githubusercontent.com/89055502/236369742-ea9bf595-37e6-4f4c-9458-89893668e463.png)
- your design
![l-d-s](https://user-images.githubusercontent.com/89055502/236369753-bc19ac1a-e6b1-409a-8917-d489a64fab3b.png)